packages/core/contracts-ovm/insured-bridge/external/AVM_CrossDomainEnabled.sol

```diff
@@ -0,0 +1,59 @@
+pragma solidity ^0.8.0;
+
+import "./iAVM_Inbox.sol";
+
+abstract contract AVM_CrossDomainEnabled {
+    modifier onlyFromCrossDomainAccount(address l1Counterpart) {
+        require(msg.sender == applyL1ToL2Alias(l1Counterpart), "ONLY_COUNTERPART_GATEWAY");
+        _;
+    }
+
+    uint160 constant offset = uint160(0x1111000000000000000000000000000000001111);
```

Comment:
nit: we should consider having a common const file for these things. this is repeated from here: https://github.com/UMAprotocol/protocol/blob/7d73fde9cf062c50a599166a0fe52f3f2a817ddb/packages/core/contracts-ovm/insured-bridge/external/arbitrum/AVM_CrossDomainEnabled.sol#L9

dont need to do this in this PR though.

Reply:
Ah I had made a local copy of `AVM_CrossDomainEnabled` to get my tests to work, will merge with your PR and remove duplicate